core/src/main/java/nl/nn/adapterframework/http/rest/ApiServiceDispatcher.java

```diff
@@ -205,11 +207,41 @@ protected JsonObject generateOpenApiJsonSchema(Collection<ApiDispatchConfig> cli
 			}
 		}
 		root.add("paths", paths.build());
-		root.add("components", Json.createObjectBuilder().add("schemas", schemas));
+		JsonObjectBuilder components = Json.createObjectBuilder();
+		components.add("schemas", schemas);
+		root.add("components", components);
 
 		return root.build();
 	}
 
+	private JsonArrayBuilder mapServers(HttpServletRequest request) {
+		JsonArrayBuilder serversArray = Json.createArrayBuilder();
+		String protocol = request.isSecure() ? "https://" : "http://";
+		String suffix = "/api";
+		String env = AppConstants.getInstance().getString("dtap.stage", "LOC");
+		int port = request.getServerPort();
+		// Get load balancer url if exists
+		String loadBalancerUrl = AppConstants.getInstance().getProperty("load.balancer.url", null);
```

Comment:
Dots are not spaces, the dot's divide categories into sub-categories; `jdbc.dateFormat` generic database related format and `jdbc.datasource.default` datasource related. I think loadbalancer should either be 1 word or have a underscore (load_balancer)